tests/e2e/forms/submit-default-delivery-form.specs.js

```diff
@@ -41,8 +42,7 @@ describe('Submit Default Delivery Report', () => {
     await deliveryReport.selectBabiesDeliveredButton();
     await deliveryReport.enterNoOfBabiesDelivered(6);
     await deliveryReport.selectBabiesAliveButton(3);
-
-    await deliveryReport.enterDeliveryDate('');
+    await deliveryReport.enterDeliveryDate(moment().format('YYYY-MM-DD'));
```

Comment:
The old date widget used to default the value to today. So, if you opened the widget and then hit `enter` or `tab` to close it, then it would store the value as today.  The new version of the widget does not behave this way. Instead if you close the widget without typing in or clicking on a value, then no value is stored for the question.